.github/CODEOWNERS

```diff
@@ -1,4 +1,16 @@
 # This file is used to auto request reviews for a pull request
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-* @autodesk/synthesis-devs
+/exporter/ @autodesk/fusion hunter.barclay@autodesk.com
+
+/fission/src/aps/ @autodesk/fusion hunter.barclay@autodesk.com
+/fission/src/mirabuf/ @autodesk/fusion hunter.barclay@autodesk.com
+/fission/src/proto/ @autodesk/fusion hunter.barclay@autodesk.com
+
```

Comment:
```suggestion

/fission/src/ui/ hunter.barclay@autodesk.com julian.wright@autodesk.com
```

Thoughts?

Reply:
I initially avoided doing this because 99% of all features are going to be editing some type of modal, this would make it so that all of those changes would request @PepperLola in addition to the normal pattern for system changes. 

If we want we can do that but it would mean that @PepperLola would need to review almost every PR. Or at least would be requested for it.